diesel/src/pg/expression/functions.rs

```diff
@@ -1423,3 +1423,38 @@ define_sql_function! {
     /// ```
     fn array_upper<Arr: ArrayOrNullableArray + SingleValue>(array: Arr, dimension: Integer) -> Nullable<Integer>;
 }
+
+define_sql_function! {
```

Comment:
This function also support a form `jsonb_object(keys text[], values text[])`, whitch takes keys and values pairwise from two separate arrays. Could you implement that too?

Reply:
yes sure ill implement that too